workflows-scripts/circulation/check-in-check-out/circulation_checkInCheckOut.jmx

```diff
@@ -3483,7 +3301,7 @@ switch (itemOrder){
                   <stringProp name="Argument.value">{&#xd;
 	&quot;itemBarcode&quot;:&quot;${item_barcode_availiable}&quot;,&#xd;
 	&quot;userBarcode&quot;:&quot;${user_barcode}&quot;,&#xd;
-	&quot;loanDate&quot;:&quot;${__time(yyyy-MM-dd&apos;T&apos;hh:mm:ssZ)}&quot;,&#xd;
+	&quot;loanDate&quot;:&quot;${__time(yyyy-MM-dd&apos;T&apos;hh:mm:ss&apos;Z&apos;)}&quot;,&#xd;
```

Comment:
Keep an eye on this change as sometimes FOLIO is tricky accepting a particular time format.  If it fails then we should use what was there.